src/containers/App.jsx

```diff
@@ -156,6 +155,22 @@ export function App(props) {
   const [renderLogout, setRenderLogout] = useState(false);
   const open = Boolean(anchorEl);
 
+  const auth = useAuth();
+  // This effect clause is needed to handle when the auth library loads the token on page refresh
+  useEffect(() => {
+    (() => {
+      if (
+        auth.isAuthenticated &&
+        !auth.isLoading &&
+        !user.isAuthenticated &&
+        !auth.activeNavigator
+      ) {
+        dispatch(logInSuccess(auth.user));
+        dispatch(getUserStatus());
+      }
+    })();
+  }, [auth, dispatch, user]);
```

Comment:
I think there is an extra layer of parenthesis in here. 
```suggestion
  useEffect(() => {
      if (
        auth.isAuthenticated &&
        !auth.isLoading &&
        !user.isAuthenticated &&
        !auth.activeNavigator
      ) {
        dispatch(logInSuccess(auth.user));
        dispatch(getUserStatus());
      }
  }, [auth, dispatch, user]);
```